modules/mbedtls/Kconfig

```diff
@@ -111,6 +112,26 @@ config MBEDTLS_DEBUG_LEVEL
 	  This makes Zephyr call mbedtls_debug_set_threshold() function during
 	  mbedTLS initialization, with the configured debug log level.
 
+config MBEDTLS_DEBUG_EXTRACT_BASENAME_AT_BUILDTIME
```

Comment:
Wouldn't it be better to make this a `choice` type configuration?

Reply:
Indeed, `choice` sounds like a better option, as having both (buildtime and runtime) enabled at the same time doesn't seem to make sense. I've added 3rd option however to disable both, just in case somebody would want that.